### ISSUE TITLE
[FORMATTING] Inline block comment has error

### ISSUE THREAD

**Issue by thanharrow** [sql-formatter-org/sql-formatter#411]
I using Postgresql support block comment.
**Input data**
/* 
/* inline block comment */
*/

**Expected Output**

```sql
/* 
/* inline block comment */
*/
```

**Actual Output**

```sql
/* 
/* inline block comment */
* /
```

It has one space character in * and /

**Usage**

- How are you calling / using the library? YES
- What SQL language(s) does this apply to? SQL
- Which SQL Formatter version are you using? 10.0.0

**Comment by nene** [comment]
Indeed. I had not realized that SQL supports nested block comments.

Will need to be fixed. Thanks for reporting.

**Comment by nene** [comment]
Did some investigation on this. Turns out that although SQL standard allows for nesting of comments, it's only supported by PostgreSQL and MS Transact-SQL.

https://github.com/sql-formatter-org/sql-formatter/wiki/Comments

**Comment by nene** [comment]
Fixed now in 10.3.0 release.
